paper.md

```diff
@@ -35,65 +35,75 @@ bibliography: paper.bib
+Git-annex provides access to a wide range of external data storage resources via various protocols, but cannot implement all idiosyncracies of any individual data portal. <!-- not clear why that would be true. likely only due to lack of interest -->
 In particular, scientific data is frequently stored in compressed archives to reduce its storage demands, or on specialized servers, such as XNAT (www.xnat.org).
 To address these demands, git-annex implemented support for external special remotes [@git-annex:special_remotes_protocol].
-This allowed DataLad and many other projects to access and provide git-annex special remotes to facilitate access to an ever growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inodes limits).
+This allowed DataLad and many other projects to access and provide git-annex special remotes to facilitate access to an ever growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inodes limits). <!-- maybe phrase as "git-annex delegates..." or use the term "plugin" to make clear that a special remote hooks directly into git-annex and is a standardization of some arbitrary 3rd-party protocol -->
 
 **They require a layer above to establish a *distribution*.**
 The DataLad project's initial goal was to provide a data distribution with unified access to the already available public data archives in neuroscience, such as http://crcns.org or http://openfmri.org.
 Git and git-annex are just "client tools" and do not provide aggregation of different resources into a unified distribution on their own.
 http://datasets.datalad.org became an example of such a data distribution curated by the DataLad team, which at the moment provides streamlined access to over 250 TBs of data across wide range of projects and archives.
+<!-- MIH thinks this needs a clarification of terms. If datasets.d.o is a distribution than datalad is "just a client tool" too. I think this is probably aimed at the nesting feature, and if so, we should use the term, because people can read about it in the handbook. This technical feature is then what is used to build a "unified data distribution"-->
 
 **Modularization is needed to scale.**
 Research workflows impose additional demands for an efficient research data management (RDM) platform besides "version control" and "data transport".
-Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single git repository even if individual files are tiny in their size.
+Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single Git repository even if individual files are tiny in their size. <!-- also Git technical limitation -->
```

Comment:
yes, and somewhat implied by mentioning `Git repository` in the sentence.  Not sure if should be acted upon, thus
```suggestion
Many research datasets contain millions of files, and that precludes placing such datasets in their entirety within a single Git repository even if individual files are tiny in their size.
```